apps/mosaico/src/Main.purs

```diff
@@ -234,7 +235,14 @@ renderArticle env uuid article mostReadArticles = do
               , "isDraft"           /\ (show $ isDraftArticle a)
               , "categoryStructure" /\ (JSON.stringify $ encodeJson env.categoryStructure)
               ]
-        appendMosaico mosaicoString env.htmlTemplate >>= appendHead (mkWindowVariables windowVars)
+            metaTags = DOM.renderToStaticMarkup $
+              DOM.fragment
+                [ DOM.meta { property: "og:type", content: "article" }
+                , DOM.meta { property: "og:title", content: _.title $ fromFullArticle a }
```

Comment:
Instead of calling `fromFullArticle a` multiple times, we could just make a variable out of it:
```
metaTags =
  let article = fromFullArticle a
  in DOM.renderToStaticMarkup $ ...
```
```

Reply:
check